### PR TITLE
fix verifying proxy setting

### PR DESCRIPTION
**Pull request by joesis**
For https://github.com/getlantern/lantern-internal/issues/920. It should report error if the proxy is not actually set.

### CODE REVIEW

File: sysproxy_test.go [hunk 1]
```diff
@@ -8,6 +8,16 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestIn(t *testing.T) {
+	assert.True(t, in("127.0.0.1:8888", "127.0.0.1:8888\n127.0.0.1:8888"))
+	assert.True(t, in("127.0.0.1:8888", "127.0.0.1:8888\n127.0.0.1:8888\n"))
+	assert.False(t, in("127.0.0.1:8888", "127.0.0.1:8888\n127.0.0.1:8887"))
```
Comment (joesis) [inline sysproxy_test.go]: Yeah I should rename the func.

**Comment by joesis** [comment]
@oxtoacart I renamed the function per your suggestion.

**Comment by oxtoacart** [comment]
LGTM.

**Comment by joesis** [comment]
Thanks @oxtoacart ! It's integrated to `flashlight` at cf540fb65abd4b69e5ca2802baf61f356876e4f5

**Pull request merged**